runtime/vara/src/tests.rs

```diff
@@ -17,116 +17,217 @@
-    check_weight_inbounds_interval!(instruction_weights, i32gtu, 2_000);
-    check_weight_inbounds_interval!(instruction_weights, i64les, 4_000);
-    check_weight_inbounds_interval!(instruction_weights, i32les, 2_000);
-    check_weight_inbounds_interval!(instruction_weights, i64leu, 4_000);
-    check_weight_inbounds_interval!(instruction_weights, i32leu, 2_000);
-
-    check_weight_inbounds_interval!(instruction_weights, i64ges, 4_000);
-    check_weight_inbounds_interval!(instruction_weights, i32ges, 2_000);
-    check_weight_inbounds_interval!(instruction_weights, i64geu, 4_000);
-    check_weight_inbounds_interval!(instruction_weights, i32geu, 2_000);
-    check_weight_inbounds_interval!(instruction_weights, i64add, 2_500);
-    check_weight_inbounds_interval!(instruction_weights, i32add, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64sub, 3_000);
-    check_weight_inbounds_interval!(instruction_weights, i32sub, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64mul, 4_000);
-    check_weight_inbounds_interval!(instruction_weights, i32mul, 2_300);
-    check_weight_inbounds_interval!(instruction_weights, i64divs, 4_800);
-    check_weight_inbounds_interval!(instruction_weights, i32divs, 3_800);
-
-    check_weight_inbounds_interval!(instruction_weights, i64divu, 5_200);
-    check_weight_inbounds_interval!(instruction_weights, i32divu, 4_200);
-    check_weight_inbounds_interval!(instruction_weights, i64rems, 21_100);
-    check_weight_inbounds_interval!(instruction_weights, i32rems, 15_100);
-    check_weight_inbounds_interval!(instruction_weights, i64remu, 4_400);
-    check_weight_inbounds_interval!(instruction_weights, i32remu, 4_300);
-    check_weight_inbounds_interval!(instruction_weights, i64and, 3_000);
-    check_weight_inbounds_interval!(instruction_weights, i32and, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64or, 3_000);
-    check_weight_inbounds_interval!(instruction_weights, i32or, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64xor, 3_000);
-    check_weight_inbounds_interval!(instruction_weights, i32xor, 1_000);
-
-    check_weight_inbounds_interval!(instruction_weights, i64shl, 2_500);
-    check_weight_inbounds_interval!(instruction_weights, i32shl, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64shrs, 2_500);
-    check_weight_inbounds_interval!(instruction_weights, i32shrs, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64shru, 2_500);
-    check_weight_inbounds_interval!(instruction_weights, i32shru, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64rotl, 2_000);
-    check_weight_inbounds_interval!(instruction_weights, i32rotl, 1_000);
-    check_weight_inbounds_interval!(instruction_weights, i64rotr, 2_500);
-    check_weight_inbounds_interval!(instruction_weights, i32rotr, 1_000);
+fn instruction_weights_heuristics_test() {
```

Comment:
furthermore, you can place such funcs with nums in runtime-common to just re-introduce three-lines test for each runtime as it's generic across `T: Config` (Runtime)

Reply:
Yes I will add todo